pkg/kube/kube_handler.go

```diff
@@ -106,9 +106,10 @@ type PinnipedConciergeConfig struct {
 
 // ClustersConfig is an alias for a map of additional cluster configs.
 type ClustersConfig struct {
-	KubeappsClusterName string
-	PinnipedProxyURL    string
-	Clusters            map[string]ClusterConfig
+	KubeappsClusterName  string
+	GlobalReposNamespace string
```

Comment:
Not sure if this is the best place to add the global repos namespace...otherwise we might need to change the register RPC method signature.

Reply:
Yeah, I think that's fine for now.